docs/book/releases/v2.1.0.md

```diff
@@ -49,14 +49,19 @@ Note: For vSphere CSI Migration feature the minimum Kubernetes version requireme
 6. When in-tree vSphere plugin is configured to use default datastore in this format default-datastore: `</datacenter-name>/datastore/<default-datastore-name>`, migration of the volume will fail as mentioned here: https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/628. If default datastore is configured in this format `<default-datastore-name>` then we do not see this issue
    - Impact: In-tree vSphere volumes will not get migrated successfully
    - Workaround:
-      - Upgrade CSI driver with this fix.
+      - Upgrade CSI driver to one of the versions which has the fix listed under `Notable Changes`
 7. When a Pod is rescheduled to a new node, there may be some lock contention which causes a delay in the volume getting detached from the old node and attached to the new node.
    - Impact: Rescheduled Pods remain in `Pending` state for an varying amount of time.
    - Workaround: Upgrade CSI driver to `v2.1.1`.
 8. When pod using a PVC is rescheduled to other node when metadatasyncer is down, fullsync might fail with error `Duplicated entity for each entity type in one cluster is found`.
    - Impact: CNS may hold stale volume metadata.
    - Workaround:
-      - Upgrade CSI driver with this fix.
+      - Upgrade CSI driver to one of the versions which has the fix listed under `Notable Changes`
+9. If there are no datastores present in any one of the datacenters in your vSphere environment, CSI file volume provisioning fails with `failed to get all the datastores` error.
+   - Impact: File volume provisioning keeps failing.
+   - Workaround:
+      - Either add a datastore to the datacenter which has no datastores on it or remove this datacenter.
+      - Upgrade CSI driver to one of the versions which has the fix listed under `Notable Changes`
```

Comment:
We dont have a release with this fix right? Maybe we should remove this line for now and add it later with the right CSI version.

Reply:
Can we remove "Upgrade CSI driver to one of the versions", from the workaround for issues, where the CSI version is not released?

Reply:
So I just say Workaround: None?

Reply:
@divyenpatel @RaunakShah Having workaround as None doesn't sound good. Should I reframe as:
```
This is being fixed in the upcoming version.
```

Reply:
means workaround is still none. we do not have a release with the fix

Reply:
we can also ask the admin to remove the read-only privilege from the unintended datacenter for the vsphere csi driver user, so it does not get listed in the query.

Reply:
> This is being fixed in the upcoming version.

This works for me too, but we'll need to remember to update that when 2.2 is released. I'll leave the final decision to you guys

Reply:
I changed it to `This is being fixed in an upcoming version` and I will track the updates for these with an internal bug for 2.2 release.